apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepTwo/StepTwoForm.tsx

```diff
@@ -62,6 +63,9 @@ const StepTwoForm: React.FC<Props> = (props) => {
             {formatMessage(rcRequestedHearingArrangements.heading)}
           </Text>
         </Box>
+        <Box component="section" marginBottom={7}>
+          <CaseInfo workingCase={workingCase} />
```

Comment:
Doesn't CaseInfo need some additional parameters?

Reply:
In this case, no. This is the screen before you choose the judge and registrar and on this screen, only the case number should be visible. If I were to pass in "userRole" and "showAdditionalInfo" props, then we would try to show the selected court without having selected a court. Guðný talked about this in a [comment on the task](https://app.asana.com/0/0/1201513921330245/1201638725706888/f)